liferay-frontend/liferay-portal
# [Only 7.4] - LPS-125192 - Remove compat user-icons

This PR removes the compat layer for user icons, so we have updated markup in some places.

![image](https://user-images.githubusercontent.com/7963804/103045400-96076a00-4584-11eb-9723-1d30373b9ceb.png)

---

To test this PR:

- Go to commerce:
  - View/Edit order notes and see user icon
- Go to Document library and add file abstract
